### ISSUE TITLE
New field added to schema is not returned by content delivery api

### ISSUE THREAD

**Issue by smohadjer** [storyblok/storyblok#682]
**The issue occurs while working with:** (check one with "x")   
- [x] *app.storyblok.com (CMS - Interface)*   
- [x] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other:* 

**Current behavior:**
When I select a content in Storyblok dashboard, then click on "define schema" and add a new field (no value set) and then save schema, and save and publish my content I do not see the new field in json that https://api.storyblok.com/v1/cdn/ returns for my content. If I set a value for new field, save and publish my content, then the field shows with value. If I remove the value again and save and publish my content, the new field still shows with empty value which is what I expected to happen the very first time I added the field too.

**Expected behavior:**
A new field should show with empty value in json returned by api as soon as it's added to schema and schema is saved/published.

**Steps to reproduce:**
Select a content, then click on "define schema" and add a new field and then save the schema, and save and publish your content. Then call https://api.storyblok.com/v1/cdn/ with parameters and inspect json returned for your content.

**Comment by gerhard50five** [comment]
Did you edit the Schema of the Component that you currently had selected or was it a nested component? If it was nested we have found that you need to view that particular component in the editor and then save, otherwise it won't update the reference.

Hope this helps.

**Comment by smohadjer** [comment]
No, it was not a nested component, I had added a field to a page component.

**Comment by gitneeraj** [comment]
Any workaround for this? 

I have 1000+ pages and its not feasible to open each and every page to save it.

**Comment by ThomasKinsmen** [comment]
Having the same issue here. Seems like there should be a better solution.

**Comment by alvarosabu** [comment]
Hi @smohadjer, we are closing this issue because this repository was used in the early days for support-related questions, but we've since transitioned to more effective channels for helping our users.

Please, refer to the following resources:

Storyblok Github Org: https://github.com/storyblok
Storyblok Docs: https://www.storyblok.com/docs
Official Support: https://support.storyblok.com
Join our Discord: https://www.storyblok.com/join-discord

Thanks for the understanding 🙏🏻
